app/api/scripts/create_eligibility_screener_csv.py

```diff
@@ -0,0 +1,153 @@
+from api.util.datetime_util import utcnow
+from api.util.string_utils import blank_for_null, join_list
+
+logger = api.logging.get_logger(__name__)
+
+DATETIME_OUTPUT_FORMAT = "%m/%d/%y %I:%M %p"
+
+
+@dataclass
+class EligiblityScreenerCsvRecord:
+    first_name: str
+    last_name: str
+    phone_number: str
+    eligibility_categories: str
+    has_prior_wic_enrollment: str
+    eligibility_programs: str
+    household_size: str
+    zip_code: str
+    chosen_wic_clinic: str
+    applicant_notes: str
+    submitted_datetime: str
+
+
+ELIGIBILITY_SCREENER_CSV_HEADER = EligiblityScreenerCsvRecord(
+    first_name="First Name",
+    last_name="Last Name",
+    phone_number="Phone Number",
+    eligibility_categories="Categorical eligibility",
+    has_prior_wic_enrollment="Have they or someone in their household been on WIC before?",
+    eligibility_programs="Adjunctive eligibility",
+    household_size="Household size (only if none of the above for Adjunctive)",  # TODO - do we need that last bit?
+    zip_code="Zip Code",
+    chosen_wic_clinic="Chosen WIC Clinic",
+    applicant_notes="Applicant notes",
+    submitted_datetime="Submitted date/time",
+)
+
+
+def create_eligibility_screener_csv(db_session: scoped_session, output_file_path: str) -> None:
+    # Get DB records
+    eligibility_screener_records = get_eligibility_screener_records(db_session)
+
+    # Convert records
+    csv_records = convert_eligibility_screener_records_for_csv(eligibility_screener_records)
+
+    # Create file
+    generate_csv_file(csv_records, output_file_path)
+
+    # Update DB records with timestamp
+    # Only update the timestamp if the file upload worked
+    update_db_record_timestamp(eligibility_screener_records)
+    db_session.commit()
+
+
+def get_eligibility_screener_records(db_session: scoped_session) -> list[EligibilityScreener]:
+    logger.info("Fetching Eligibility Screener records from DB")
+    eligibility_screener_records = (
+        db_session.query(EligibilityScreener)
+        .filter(EligibilityScreener.added_to_eligibility_screener_at == None)  # noqa: E711
+        .all()
+    )
+
+    record_count = len(eligibility_screener_records)
+    logger.info(
+        "Found %s eligibility screener records",
+        record_count,
+        extra={"eligibility_screener_record_count": record_count},
+    )
+
+    if record_count == 0:
+        logger.warning("No eligibility screener records found that weren't already added to a CSV")
+
+    return eligibility_screener_records
+
+
+def convert_eligibility_screener_records_for_csv(
+    records: list[EligibilityScreener],
+) -> list[EligiblityScreenerCsvRecord]:
+    logger.info("Converting eligibility screeners to CSV format")
+    out_records: list[EligiblityScreenerCsvRecord] = []
+    out_records.append(ELIGIBILITY_SCREENER_CSV_HEADER)
+
+    for record in records:
+        # TODO - do we want to convert this timestamp to a different timezone beside UTC?
```

Comment:
Oh great question. I think yes. We should set this to CT. Can we make this an env var?

Reply:
Yes, went ahead and added a utility for converting datetime objects between timezones while I was at it.